custom/ewsghana/reports/email_reports.py

```diff
@@ -54,8 +54,8 @@ def _stock_status(status, loc):
 
         locations = self.get_locations(self.config['location_id'], self.config['domain'])
 
-        products = self.unique_products(locations)
-        row_data = {product.name: defaultdict(lambda: 0) for product in products}
+        row_data = defaultdict(lambda: {'total_fac': 0, 'reported_fac': 0,
+                                        'stockout': 0, 'low': 0, 'overstock': 0, 'adequate': 0})
```

Comment:
Two questions:
- It looks like the behavior might be slightly different now since there might be rows missing from the report if a product is not added to `row_data` in the for loop. Is that intentional?
- Why was there a mismatch between the pre-polution of products in `row_data` and the actual products that were being found in the for loop? Is that indicative of a bigger issue?